.github/workflows/pre-release.yml

```diff
@@ -28,14 +28,26 @@ jobs:
           title: "Development build"
           files: package/*
 
-      - name: "Prepare ontology files for commit"
+      - name: Prepare files to be committed to another repository
         run: |
           mkdir output_files
           gzip -cd package/opencs.ttl.gz > output_files/opencs.ttl
           gzip -cd package/opencs.rdf.gz > output_files/opencs.rdf
           gzip -cd package/opencs.nt.gz > output_files/opencs.nt
 
-      - name: "Push the files to the Github Pages repository"
+      - name: "Install required packages for Python"
+        run: python -m pip install pyyaml PyGithub
```

Comment:
I think I already made a relevant comment about this – but the packages should be installed by Pipenv, when the ci-worker's image is built. It makes no sense to download these packages every time we run the pipeline.